pkg/sqlite/load.go

```diff
@@ -254,6 +256,12 @@ func (s *SQLLoader) AddPackageChannels(manifest registry.PackageManifest) error
 				return fmt.Errorf("%s specifies replacement that couldn't be found", c.CurrentCSVName)
 			}
 
+			// If we find 'replaces' in the circuit list then we've seen it already, break out
+			if _, ok := replaceCircuit[replaces]; ok {
+			   return fmt.Errorf("Circuit detected, %s replaces %s", channelEntryCSVName, replaces)
+			}
+			replaceCircuit[channelEntryCSVName] = true
```

Comment:
this should actually be "replaces", it still works but it will cause an extra time around the loop.

Reply:
done